### PR TITLE
Vol2 hpo final

### PR DESCRIPTION
**Pull request by mseeger**
*Description of changes:*

- Less log output for `tuner.run()`
- Added discussions at end of notebooks, except for `sh-async.md`

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mseeger** [comment]
Also this fixes a bug in the `lenet.md` notebook (not introduced by me)

**Comment by mseeger** [comment]
Damn, this tests all existing notebooks? Why, I did only change `lenet.md` and the ones in the HPO chapter ...

**Comment by AnirudhDagar** [comment]
I think @astonzhang cleared cache last night after making the vol2 merges; that's why it needs to rerun everything in MxNet. That bug in lenet swept in because vol2-hpo merge PR was run without clearing the cache first, and CI never checked LeNet. Maybe it came through some HPO branch edits/conflict resolution. So it is OK to ensure everything is running, given the inter-dependence of all the notebooks.

Last night we also had some issues with our infra; we are now running on 3 GPUs. One of the V100s just died (possibly a hardware thing, which is very rare, happened once before 6-7 months ago) on the EC2 instance. So it's going to be a bit slow; apologies for that.

**Comment by mseeger** [comment]
Sure. I continue to add some exercises.

**Comment by mseeger** [comment]
I don't understand why the CI failed. In `rs-async.md`, Syne Tune reported that a trial failed (actually, the very first one). But the very same criterion function is used in `sh-async.md`, which runs through fine. Also, I did not change this code.

File: chapter_hyperparameter_optimization/sh-async.md [hunk 1]
```diff
@@ -229,3 +231,8 @@ configurations as quickly as possible to the next rung level, even if this means
 promoting some wrong ones. In practice, this usually does not hurt much, and the
 gains of asynchronous versus synchronous scheduling are usually much higher
 than the loss of the suboptimal decision making.
+
+
+:begin_tab:`pytorch`
+[Discussions](https://discuss.d2l.ai/t/asynchronous-successive-halving/???)
```
Comment (astonzhang) [inline chapter_hyperparameter_optimization/sh-async.md:237]: Need this ID

```suggestion
[Discussions](https://discuss.d2l.ai/t/???)
```

File: chapter_hyperparameter_optimization/sh-intro.md [hunk 2]
```diff
@@ -262,3 +262,8 @@ computation of the HPO instead of just reducing the wall-clock time.
 
 We implemented and evaluated successive halving, a simple yet efficient
 multi-fidelity HPO algorithm.
+
+
+:begin_tab:`pytorch`
+[Discussions](https://discuss.d2l.ai/t/multi-fidelity-hyperparameter-optimization/12094)
```
Comment (astonzhang) [inline chapter_hyperparameter_optimization/sh-intro.md:268]:
```suggestion
[Discussions](https://discuss.d2l.ai/t/12094)
```

**Comment by mseeger** [comment]
@astonzhang Until when do you still accept exercises for the new release?

**Comment by mseeger** [comment]
The exercises do not format correctly. I'll open another PR.

**Comment by astonzhang** [comment]
> @astonzhang Until when do you still accept exercises for the new release?

For this run:
Accepting any non-code-related changes till Dec 11 Sunday 11:59p (your time zone) 
Accepting any code-related changes till Dec 10 Saturday 11:59p (your time zone)

**Pull request merged**